src/@types/index.ts

```diff
@@ -1382,22 +1382,24 @@ export namespace RiotAPITypes {
     }
 
     export interface LeagueEntryDTO {
-      leagueId: string;
+      leagueId?: string;
       /** Player's encrypted summonerId. */
       summonerId: string;
       summonerName: string;
       queueType: string;
-      tier: string;
-      rank: string;
-      leaguePoints: number;
+      ratedTier?: 'ORANGE' | 'PURPLE' | 'BLUE' | 'GREEN' | 'GRAY';
```

Comment:
From the docs – https://developer.riotgames.com/apis#tft-league-v1/GET_getLeagueEntriesForSummoner

> (Legal values: ORANGE, PURPLE, BLUE, GREEN, GRAY)

Reply:
So we should have rated tier already created as a type that you can re-use: https://github.com/fightmegg/riot-api/pull/441/files/437905f1ee5db9a51e6f9a9ee99b53543f01ea5a#diff-ff1ea3328dcbd401402ff9392e3dcf0d02f6cc30f1782d1344a8f6fbcb58f9cbL1345

Reply:
Fixed